tests/functional/test_get_token.py

```diff
@@ -0,0 +1,43 @@
+from flask import json
+
+
+def test_get_token(client, new_user):
```

Comment:
Love the tests 👍

Reply:
You could consider adding a test case where you try to post something with an expired token and making you sure you return an understandable response in that case

But in the setup for that test you'll have to create an expired token - which is kind of weird. 
Hard coding that token in the test is not a good idea as your environments might have different JWT secrets, so you'll still need to actually create it during the test 😅